### DISCUSSION TITLE
CIPP Automated Phishing Feature

### DISCUSSION THREAD

**Discussion by MAP360** [KelvinTegelaar/CIPP#2067] [General]
I am currently going through the testing phase of the new Phishing feature that was released and was wondering if there was a method to test and show the client what will happen when they are being redirected to another URL instead of login.microsoftonline.com

Is there a way that I can modify the URL for demo purposes to show what it should look like and what it would look like when a Phishing attempt has occurred? I have seen the short video of how it will look, but looking for something that you can demo to a client before implementation.

**Comment by rvdwegen** [comment]
You'll have to setup evilnginx.

**Reply by MAP360** [reply]
Thanks for your feedback,
